airflow/configuration.py

```diff
@@ -580,10 +589,24 @@ def getint(self, section, key):
     def getfloat(self, section, key):
         return float(self.get(section, key))
 
+    def remove_option(self, section, option):
```

Comment:
The defaults section is kind of messy but, presumably, if someone wants to remove something from defaults, they can use `self.defaults.remove`